test/tests/benchmark/runtime_tests.py

```diff
@@ -0,0 +1,118 @@
+from config.benchmark_config import *
+from config.amqp import *
+from modules.logger import Log
+from modules.amqp import AMQPWorker
+from on_http import NodesApi as Nodes
+from on_http import WorkflowsApi as Workflows
+from on_http import rest
+from datetime import datetime
+from proboscis.asserts import assert_equal
+from proboscis.asserts import assert_false
+from proboscis.asserts import assert_raises
+from proboscis.asserts import assert_not_equal
+from proboscis.asserts import assert_is_not_none
+from proboscis.asserts import assert_true
+from proboscis import SkipTest
+from proboscis import test
+from json import loads
+
+from ansible_control import ansibleControl
+import json
+import time
+
+from data_display import DataDisplay
+
+LOG = Log(__name__)
+
+@test(groups=['runtime.tests'])
+class RuntimeTests(object):
```

Comment:
How come these are reimplementing tests already defined in the other test groups? We should be running those.